src/main/resources/io/jenkins/plugins/gitlabbranchsource/WebhookRegistrationTrait/config.jelly

```diff
@@ -1,6 +1,10 @@
+
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
-    <f:entry title="${Strategy}" field="mode">
+    <f:entry title="${%Enable System Hooks}" field="isAdmin">
```

Comment:
I assume this shows up under server config ie. System configuration and not the job config.

Reply:
Okay 👍. I will add it to the server config and let user override the system hook with this trait similar to the webhook.